File: coffee.rb
class Coffee
    def initialize(temp = 0, flavor = 'bland')
        @temperature = temp
        @flavor = flavor
    end

    def temp!(temp)
        @temperature = 
        temp
    end

    def hot?(temp)
        if temp > 160
            return true
        end

        false
    end

    def display_temp
        puts @temperature
    end
end

c = Coffee.new(90)
puts c.hot?(170)

File: 2018/python/coffee.py
class Coffee:
    def __init__(self, temp=0, flavor='bland'):
        self.temperature = temp
        self.flavor = flavor

    def change_temp(self, temp):
        self.temperature = temp

    def is_hot(self, temp):
        if temp > 160:
            return True
        else:
            return False

    def display_temp(self):
        print(self.temperature)


c = Coffee(90)
print(c.is_hot(170))
